<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Instance</class>
 <widget class="QWidget" name="Instance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="name_label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Instance Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="customer_name_LineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="clientIdLabel">
       <property name="text">
        <string>API ID:</string>
       </property>
       <property name="buddy">
        <cstring>customer_name_LineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="api_id_LineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="clientSecretLabel">
       <property name="text">
        <string>API Secret Key:</string>
       </property>
       <property name="buddy">
        <cstring>api_id_LineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="api_secret_LineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="env_label">
       <property name="text">
        <string>Environment:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="env_comboBox">
       <property name="currentIndex">
        <number>5</number>
       </property>
       <item>
        <property name="text">
         <string>EAP01 (Beta)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prod01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prod02</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prod03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prod04</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prod05 (US)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prod06 (Europe)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prodnrt (Japan)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prodsyd (Australia)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prodUK (Not supported)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AWS GovCloud US (Not supported)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="org_key_LineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="org_label">
       <property name="text">
        <string>Org Key:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_invalid">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>41</green>
           <blue>41</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>41</green>
           <blue>41</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>190</red>
           <green>190</green>
           <blue>190</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ef2929;&quot;&gt;Invalid entry, this instance will not be saved&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
